_data/tasks.yml

```diff
@@ -23,4 +23,7 @@ toc:
   section:
   - title: Assigning Pods to Nodes
     path: /docs/tasks/administer-cluster/assign-pods-nodes/
-
```

Comment:
@janetkuo Can we do it all at once, after we finish all the tasks? or do you think this PR should contain that change as well?